cypress/integration/add_record_childred_admin.ts

```diff
@@ -0,0 +1,41 @@
+describe('Adult Group', () => {
+  beforeEach(() => {
+    cy.getCookies().should('be.empty');
+    cy.setCookie('hackneyToken', Cypress.env('TEST_KEY_CHILDREN_GROUP'));
+    cy.getCookie('hackneyToken').should(
+      'have.property',
+      'value',
+      Cypress.env('TEST_KEY_CHILDREN_GROUP')
+    );
+    cy.visit(Cypress.env('HOST') + '/search');
+  });
+
+  it('Confirm person type is adult', () => {
```

Comment:
Same as above, probably it would be more useful accessing directly the person page and check that the `case-notes-recording` is working properly.